Assets/Scripts/WorldRotation/WorldRotator.cs

```diff
@@ -8,22 +8,22 @@ namespace UntitledCube.WorldRotation
     public class WorldRotator : MonoBehaviour
     {
         [Header ("Rotation Testing")]
-        [SerializeField] private List<RotationDirection> _rotationDirections = new();
+        [SerializeField] private List<WorldRotations> _rotationDirections = new();
         [SerializeField] private float _timebetweenRotations = 2f;
 
         [Header("Settings")]
         [SerializeField] private float _stepAmount = 0.01f;
         [SerializeField] private float _betweenSteps = 0.01f;
 
-        private Dictionary<RotationDirection, Vector3> _directions = new Dictionary<RotationDirection, Vector3>()
+        private Dictionary<WorldRotations, Vector3> _directions = new Dictionary<WorldRotations, Vector3>()
         {
-            { RotationDirection.UP, new Vector3(-90f, 0f, 0f) },
-            { RotationDirection.DOWN, new Vector3(90f, 0f, 0f) },
-            { RotationDirection.LEFT, new Vector3(0f, 90f, 0f) },
-            { RotationDirection.RIGHT, new Vector3(0f, -90f, 0f) }
+            { WorldRotations.UP, new Vector3(-90f, 0f, 0f) },
+            { WorldRotations.DOWN, new Vector3(90f, 0f, 0f) },
+            { WorldRotations.LEFT, new Vector3(0f, -90f, 0f) },
```

Comment:
```suggestion
            { WorldRotations.LEFT, new Vector3(0f, 270f, 0f) },
```
Nit pick but I personaly prefer not using minus rotations. Do please check if it still works